inference-engine/samples/speech_sample/CMakeLists.txt

```diff
@@ -2,7 +2,11 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+file (GLOB SRC ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp
+file (GLOB HDR ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
+               ${CMAKE_CURRENT_SOURCE_DIR}/*.h)
+
 ie_add_sample(NAME speech_sample
-              SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/main.cpp"
-              HEADERS "${CMAKE_CURRENT_SOURCE_DIR}/speech_sample.hpp"
-              DEPENDENCIES ie_samples_utils)
+              SOURCES ${SRC}
+              HEADERS ${HDR}
+              DEPENDENCIES cnpy ie_samples_utils)
```

Comment:
I don't see where we include `cnpy` in for this sample

How will it work out of OpenVINO release package? 

Also we don't install cnpy and zlib during make install

Reply:
@ilya-lavrenov , please, look. I changed inference-engine/CMakeLists.txt   and inference-engine/samples/CMakeLists.txt .  Now it works good , if we run make install. I checked it, I ran make install and then in output directory of make install ran speech sample, and it works and use cnpy library.